src/Materials/Textures/baseTexture.ts

```diff
@@ -9,6 +9,7 @@ import { Constants } from "../../Engines/constants";
 import { IAnimatable } from '../../Animations/animatable.interface';
 import { GUID } from '../../Misc/guid';
 import { ISize, Size } from '../../Maths/math.size';
+import { HDRFiltering } from "./Filtering/hdrFiltering";

```

Comment:
Actually this could be placed in Engine.Constants and reuse from constants here to prevent the coupling and bundle size if not in use :-)